vald/evm/evm.go

```diff
@@ -471,32 +472,34 @@ func (mgr Mgr) getTxReceiptIfFinalized(chain nexus.ChainName, txID common.Hash,
 	}
 
 	txReceipt, err := client.TransactionReceipt(context.Background(), txID)
+	keyvals := []interface{}{"chain", chain.String(), "tx_ID", txID.Hex(), "conf_height", confHeight}
+	logger := mgr.logger.With(keyvals...)
 	if err == ethereum.NotFound {
-		mgr.logger.Debug(fmt.Sprintf("transaction receipt %s not found", txID.Hex()))
+		logger.Debug(fmt.Sprintf("transaction receipt %s not found", txID.Hex()))
 		return nil, nil
 	}
 	if err != nil {
-		return nil, sdkerrors.Wrap(err, "failed getting transaction receipt")
+		return nil, sdkerrors.Wrap(errors.With(err, keyvals...), "failed getting transaction receipt")
```

Comment:
Should add the tx hash in hex here as a key val

Reply:
I don't follow, it is part of the keyvals

Reply:
Ah, nvm, I misread it